packages/react-i18n/src/i18n.ts

```diff
@@ -495,22 +499,30 @@ export class I18n {
     return decimal.length === 0 ? DECIMAL_NOT_SUPPORTED : decimal;
   }
 
-  private numberDecimalSymbol() {
-    return this.formatNumber(1.1, {
+  private numberSymbols() {
+    const formattedNumber = this.formatNumber(123456.789, {
```

Comment:
```suggestion
    const formattedNumber = this.formatNumber(123456.7, {
```

Do you need 3 numbers after the decimal separator? since `maximumFractionDigits` and `maximumFractionDigits` are both `1`.

Reply:
Good point, I'll change it to your suggestion